nbbuild/default-properties.xml

```diff
@@ -66,8 +66,8 @@
 
   <property name="test.dist.dir" location="${nb.build.dir}/testdist"/>
   
-  <property name="binaries.cache" location="${user.home}/.hgexternalcache"/>
-  <property name="binaries.server" value="https://netbeans.osuosl.org/binaries/"/>
+  <property name="binaries.cache" location="${user.home}/.netbeans/nb_external_cache"/>
+  <property name="binaries.server" value="http://netbeans.osuosl.org/binaries/"/>
```

Comment:
Is the switch from `https` to `http` intentional? If so, why?

Reply:
It is not intentional.. It was very late last night after getting the kids to bed, I could have done that by accident.

It is now fixed..